numpy/core/src/umath/ufunc_object.c

```diff
@@ -3144,7 +3489,7 @@ PyUFunc_GenericFunction(PyUFuncObject *ufunc,
 
     nin = ufunc->nin;
     nout = ufunc->nout;
-    nop = nin + nout;
+    nop = ufunc->nargs;
```

Comment:
Any particular reason for this change? I'd be inclined to deprecate `nargs`, we already have a comment next to it's definition saying "why does this exist"

Reply:
No... I honestly do not care either way if we have it around or not. `nop` is used quite a lot all over, I guess. Usage is seems a bit mixed, some use nargs, some use `nin + nout`.